app/CodeFormatter/PhpCodeFormatter.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace DevCommunityDE\CodeFormatter\CodeFormatter;
+
+/**
+ * Class PhpCodeFormatter.
+ */
+class PhpCodeFormatter extends CodeFormatter
+{
+    protected const LANGUAGES = [
+        'php',
+    ];
+
+    /**
+     * @return string
+     */
+    protected function getShellCommand(): string
+    {
+        return 'php-cs-fixer fix --rules @PSR2 -';
```

Comment:
Lets try this flag:
```suggestion
        return 'php-cs-fixer fix --rules @PSR2 --quiet -';
```

And if this is not enough, we can redirect stderr to /dev/null

```suggestion
        return 'php-cs-fixer fix --rules @PSR2 --quiet - 2>/dev/null';
```